operators/src/upgrade/controller/reconciler.rs

```diff
@@ -235,7 +260,28 @@ impl ResourceContext {
                 debug!(?status, "Deleted collection of pods: status");
             }
         }
-        Ok(Action::await_change())
+        debug!("Deleted io-engine Pods");
+
+        debug!("Waiting for new io-engine Pods to start...");
```

Comment:
We're deleting all io-engine pods at once here, which means all volumes will be broken. Should we check that there are no published volumes?